randovania/generator/old_generator_reach.py

```diff
@@ -103,10 +103,11 @@ def reach_from_state(
         return reach
 
     def _potential_nodes_from(self, node: Node) -> Iterator[tuple[Node, RequirementSet]]:
-        extra_requirement = _extra_requirement_for_node(self._game, self.node_context(), node)
-        requirement_to_leave = node.requirement_to_leave(self._state.node_context())
+        context = self._state.node_context()
```

Comment:
Welcome to one of the hottest code paths in Randovania! I did this directly here because a function call is slow in python. Probably could do the same elsewhere in the function.